pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -166,28 +168,67 @@ var _ = Describe("Tuned", func() {
 			When("realtime workload enabled", func() {
 				It("should not contain idle=poll cmdline", func() {
 					profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
-						RealTime:             pointer.BoolPtr(false),
-						HighPowerConsumption: pointer.BoolPtr(false),
+						HighPowerConsumption: pointer.BoolPtr(true),
+						RealTime:             pointer.BoolPtr(true),
 					}
 					tunedData := getTunedStructuredData(profile)
 					bootLoader, err := tunedData.GetSection("bootloader")
 					Expect(err).ToNot(HaveOccurred())
-					Expect(bootLoader.Key("cmdline_idle_poll").String()).ToNot(Equal(cmdlineIdlePoll))
+					Expect(bootLoader.Key("cmdline_idle_poll").String()).To(Equal(cmdlineIdlePoll))
 				})
 			})
 
 			When("realtime workload disabled", func() {
 				It("should contain idle=poll cmdline", func() {
 					profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
 						HighPowerConsumption: pointer.BoolPtr(true),
-						RealTime:             pointer.BoolPtr(true),
+						RealTime:             pointer.BoolPtr(false),
 					}
 					tunedData := getTunedStructuredData(profile)
 					bootLoader, err := tunedData.GetSection("bootloader")
 					Expect(err).ToNot(HaveOccurred())
-					Expect(bootLoader.Key("cmdline_idle_poll").String()).To(Equal(cmdlineIdlePoll))
+					Expect(bootLoader.Key("cmdline_idle_poll").String()).ToNot(Equal(cmdlineIdlePoll))
 				})
 			})
+
+			When("perPodPowerManagement Hint to true", func() {
+				It("should fail as PerPodPowerManagement and HighPowerConsumption can not be set to true", func() {
+					profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+						HighPowerConsumption:  pointer.BoolPtr(true),
+						PerPodPowerManagement: pointer.BoolPtr(true),
+					}
+					_, err := NewNodePerformance(profile)
+					Expect(err).To(HaveOccurred())
```

Comment:
do we want to test if the error we get actually means the two hints are conflicting? otherwise we can get unrelated errors and misinterpret it. Can be done later if requires too deep changes.

Reply:
good catch, thx. Done